Adding first version of the GUI.

davidevaleriani opened the pull request:
Adding first implementation of the GUI with the various components in place.

Commit:
Adding first implementation of the GUI with the various components in place.

monchote commented on app/src/main/AndroidManifest.xml:
```diff
@@ -23,6 +23,14 @@
+            <intent-filter>
+                <action android:name="android.intent.action.MAIN" />
+
+                <category android:name="android.intent.category.LAUNCHER" />
```
Do we want 2 launcher main activities? Is the idea to have one for portrait and one for landscape?

davidevaleriani replied:
No, I added it there temporarily as suggested by Javi because at some point we need to perform a merging of the mainActivity and the UIActivity.
The UI is only landscape to optimise the design.

monchote replied:
Cool, feel free to remove the other one Launcher activity from here now and I'll merge my code into the new one at some point.

monchote commented on app/src/main/java/uk/co/neuralcubes/neuralates/ControlFragment.java:
```diff
@@ -0,0 +1,96 @@
+public class ControlFragment extends Fragment {
+    private Spinner selectSphero, selectMuse;
+    private TextView batterySphero, batteryMuse;
+    private Button fp1, fp2, tp9, tp10;
```
Can we use a List<Button> here instead?

davidevaleriani replied:
Do you mean a list of buttons for the electrodes? Sure but I thought it was easier to identify which electrode is which from the name.

monchote replied:
The order can be determined by an array resource of ids. It shouldn't be an issue.

Commit:
Removed MainActivity from the manifest, so that we only have the UIActivity.
Transformed the four variables representing the electrodes in a List.
Fixed some bugs around.
